src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -446,7 +450,8 @@ extension RiskProvider {
 				mostRecentDateWithHighRisk: risk.details.mostRecentDateWithRiskLevel,
 				numberOfDaysWithLowRisk: risk.details.numberOfDaysWithRiskLevel,
 				numberOfDaysWithHighRisk: risk.details.numberOfDaysWithRiskLevel,
-				calculationDate: Date()
+				calculationDate: Date(),
+				riskLevelPerDate: [Date(): .low]
```

Comment:
Shouldn't this be 
```suggestion
				riskLevelPerDate: [Date(): .high]
```
?

Reply:
Good finding - I'll change it